### PR TITLE
fix(navigation): nagivate to profile/_update endpoint

### PR DESCRIPTION
**Pull request by corinnekrych**
fixes https://github.com/openshiftio/openshift.io/issues/2124

### CODE REVIEW

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2470-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2470-fabric8-ui.badger.fabric8.io

**Comment by bartoszmajsak** [comment]
Is it a new way of titling PRs? :) I would rather expect something like `fix(...): ......`

File: src/app/shared/owner-guard.service.spec.ts [hunk 1]
```diff
@@ -50,10 +50,48 @@ describe('OwnerGuard', () => {
...
+      fragment: null,
+      params: null,
+      data: null,
+      outlet: null,
+      component: null,
+      routeConfig: null,
+      parent: null,
+      paramMap: null,
+      queryParamMap: null,
+      firstChild: null
```
Comment (bartoszmajsak) [inline src/app/shared/owner-guard.service.spec.ts]: Is it really necessary to have all these `null`s explicitly set?

**Comment by bartoszmajsak** [comment]
I cannot get it to build on my machine (latest master works though).

Here's the error I'm facing. https://gist.github.com/11627de4fc445fcd9290106715564037

**Comment by corinnekrych** [comment]
@bartoszmajsak wierd badger indeed, let me rebase and refresh badger

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2470-3 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2470-fabric8-ui.badger.fabric8.io

**Comment by corinnekrych** [comment]
@bartoszmajsak it looks better to me:
feature-flag-navigate-profile

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2470-4 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2470-fabric8-ui.badger.fabric8.io

**Comment by joshuawilson** [comment]
What does this fix? It looks like it works in prod. 
When I am set to view only released level and then go to the Apps page and click the opt-in button, it takes me to the profile page.

**Comment by corinnekrych** [comment]
@joshua this issue was when you are logged in and you try to access some planner in some spce you do not own. The owner-guard ussd to cancel the navigation when hitting ‘go to profile’ button.

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2470-6 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2470-fabric8-ui.badger.fabric8.io

**Pull request merged**